tests/phpunit/Presentation/ListFacetHtmlBuilderTest.php

```diff
@@ -48,15 +56,33 @@ private function newListFacetHtmlBuilder(): ListFacetHtmlBuilder {
 		);
 	}
 
+	public function testCheckboxesViewModelContainsLabels(): void {
+		$viewModel = $this->buildViewModel();
+
+		$this->assertArrayHasKey( 'checkboxes', $viewModel );
+		$this->assertArrayHasKey( 'msg-show-more', $viewModel['checkboxes'] );
+		$this->assertArrayHasKey( 'msg-show-less', $viewModel['checkboxes'] );
+		$this->assertIsString( $viewModel['checkboxes']['msg-show-more'] );
+		$this->assertIsString( $viewModel['checkboxes']['msg-show-less'] );
+	}
```

Comment:
This type of test adds little value because it is not testing logic. It is testing wiring, and doing so directly, on the unit test level. If the wiring is wrong, like using the wrong key, the test won't notice. An integration test that verifies the message is in the rendered template would provide some value, though it's not clear such a test is called for, certainly not if we are talking about multiple such tests for a single template rather than a smoke test.